Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG BUILD_IMAGE=/weaveworks/eksctl-build:0849e149eca6668d7e1a5b400dbbbb5cac650e52
+FROM $BUILD_IMAGE as build
```

Comment:
Do we need the build image as base or can we just use a distroless?

Reply:
the final image is distroless:
https://github.com/weaveworks/eksctl/pull/2511/files#diff-3254677a7917c6c01f55212f86c57fbfR19